.github/workflows/changeset-snapshot/test-aperture.js

```diff
@@ -9,7 +9,7 @@ module.exports = async ({ github, context }, execa) => {
     const newTags = Array
         .from(stdout.matchAll(/New tag:\s+([^\s\n]+)/g))
         .map(([_, tag]) => tag)
-        .filter((tag) => !/^wc-.+$|pie-(monorepo|docs|storybook)/.test(tag));
+        .filter((tag) => !/^wc-.+$|pie-(monorepo|docs|storybook|git-hooks-scripts|webc-core|webc-testing|icons-configs|wrapper-react)|generator-pie-component/.test(tag));
```

Comment:
Its fine for now but i think this is a bit brittle (like we discussed yesterday for the web-c automation script). Do you have any ideas on how we could avoid having to remember to update this list?

Reply:
Completely agree. As discussed yesterday, I think adding an extra property to the pieMetadata in package.json would probably make the most sense. We could also take another look what which packages are private/public, but there's a risk this will have a knock-on effect with package resolutions.